docs/en/sql-reference/aggregate-functions/reference/stochasticlinearregression.md

```diff
@@ -72,5 +72,5 @@ The query will return a column of predicted values. Note that first argument of
 
 **See Also**
 
-- [stochasticLogisticRegression](../../../sql-reference/aggregate-functions/reference/stochasticlogisticregression.md#stochasticlogisticregression)
+- [stochasticLogisticRegression](../../../sql-reference/aggregate-functions/reference/stochasticlogisticregression.md)
```

Comment:
I am not sure about the relative path here and in other places

Reply:
It works correctly, but could probably be simplified.

I removed the commit with markdown changes, because it got fixed in https://github.com/ClickHouse/ClickHouse/pull/74650